app/controllers/follow_requests_controller.rb

```diff
@@ -3,26 +3,31 @@ class FollowRequestsController < ApplicationController
 
   # GET /follow_requests or /follow_requests.json
   def index
+    authorize @follow_request
```

Comment:
if you refactor to use before_action {authorize ...}, you can remove each of these authorize @ follow_request